synapse/handlers/stats.py

```diff
@@ -110,10 +122,10 @@ async def _unsafe_process(self):
             )
 
             for room_id, fields in room_count.items():
-                room_deltas.setdefault(room_id, {}).update(fields)
+                room_deltas.setdefault(room_id, Counter()).update(fields)
 
             for user_id, fields in user_count.items():
-                user_deltas.setdefault(user_id, {}).update(fields)
+                user_deltas.setdefault(user_id, Counter()).update(fields)
```

Comment:
I think this is fine because `fields` is a `Dict[str, int]`, which in this case is effectively the same as `Counter[str]`?

Reply:
Pretty much, the alternative would be to make `user_deltas` a `Dict[str, Dict[str, int]]` instead of a `Dict[str, Counter[str]]` I think? Would that be better?

Reply:
Nah, it's fine, just wanted to check my understanding of the change was correct :)